tips/TIP-0029/tip-0029.md

```diff
@@ -0,0 +1,254 @@
+                <blockquote>Defines dynamic changes to the PoW parameters.</blockquote>
+              </details>
+              <details>
+                <summary>Receipts Milestone Option</summary>
+                <blockquote>Defines UTXOs for newly migrated funds.</blockquote>
+              </details>
+            </td>
+          </tr>
+        </table>
+      </details>
+    </td>
+  </tr>
+  <tr>
+    <td>Signatures Count</td>
+    <td>uint8</td>
+    <td>The number of signature entries following.</td>
+  </tr>
+  <tr>
+    <td valign="top">Signatures <code>anyOf</code></td>
+    <td colspan="2">
+      <details open="true">
+        <summary>Ed25519 Signature</summary>
+        <table>
+          <tr>
+            <th>Name</th>
+            <th>Type</th>
+            <th>Description</th>
+          </tr>
+          <tr>
+            <td>Signature Type</td>
+            <td>uint8</td>
+            <td>
+              Set to <strong>value 0</strong> to denote an <i>Ed25519 Signature</i>.
+            </td>
+          </tr>
+          <tr>
+            <td>Public Key</td>
+            <td>ByteArray[32]</td>
+            <td>The Ed25519 public key of the signature.</td>
+          </tr>
+          <tr>
+            <td>Signature</td>
+            <td>ByteArray[64]</td>
+            <td>The Ed25519 signature signing the BLAKE2b-256 hash of the serialized <i>Milestone Essence</i>.</td>
+          </tr>
+        </table>
+      </details>
+    </td>
+  </tr>
+</table>
+
+### Milestone options
+
+The `Options` field holds additional data authenticated by the milestone.
+
+The following table lists all the `Milestone Option Type` that are currently supported as well as links to the corresponding specification:
+
+| Payload Name | Type Value | TIP                                                        |
+| ------------ | ---------- | ---------------------------------------------------------- |
+| Receipt      | 0          | [TIP-34](../TIP-0034/tip-0034.md#receipt-milestone-option) |
+| PoW          | 1          | [TIP-29](#pow-milestone-option)                            |
+
+#### PoW Milestone Option
+
+The _PoW Milestone Option_ allows to change the PoW protocol parameters. 
+
+| Name                           | Type   | Description                                                                                           |
+| ------------------------------ | ------ | ----------------------------------------------------------------------------------------------------- |
+| Milestone Option Type          | uint8  | Set to **value 1** to denote a *PoW Milestone Option*.                                                |
+| Next PoW Score                 | uint32 | The new PoW score all messages should adhere to.                                                      |
+| Next PoW Score Milestone Index | uint32 | The index of the first milestone that will require a new minimal pow score for applying transactions. |
```

Comment:
There used to be syntactic rules associated with this, is this not the case anymore ?

Reply:
Actually, there are even more things that need to be clarified in this context, e.g. when exactly is a message with too low PoW invalid and where is it rejected from...
Because of this, I've removed the paragraph completely it will be re-added in the upcoming Dynamic PoW TIP